source/MRViewer/MRViewer.cpp

```diff
@@ -939,6 +939,8 @@ bool Viewer::loadFiles( const std::vector< std::filesystem::path>& filesList )
                 else
                 {
                     std::string historyName = loadedObjects.size() == 1 ? "Open file" : "Open files";
+                    if ( globalHistoryStore_ )
+                        globalHistoryStore_->nextIsLoadingFiles();
```

Comment:
I think the fix must be much simpler: just call `setSavedState()` here if the scene was empty before opening